src/handwritten/_install_daqmx.py

```diff
@@ -80,7 +79,7 @@ def _is_latest_version_greater(latest_version : str, installed_version: str) ->
         if latest_parts > installed_parts:
             return True
         else:
-            print(f"Latest Version already installed")
+            print(f"Installed NI-DAQmx version ({installed_version}) is equivalent or newer than the known version to install, {latest_parts}.")
```

Comment:
This message is extra wordy in a way that highlights a potential problem.

What is the significance of "the known version to install"? You hardcoded this version in a file in the nidaqmx-python driver. That made this version number known to the nidaqmx-python install script.

Are you and/or the Venus team planning to update this hardcoded file for each new version of NI-DAQmx? Does that include patches and minor versions?

Does this update policy align with the one used for NI Update Service on Windows?

In the long run, should this metadata be on ni.com instead?

Reply:
@bkeryan, in the previous review comment @zhindes recommended to replace the message "Latest Version already installed" with "
Installed NI-DAQmx version (23.8.0) is equivalent or newer than the known version to install, (23, 8, 0)".
@zhindes your thoughts on this ?

Are you and/or the Venus team planning to update this hardcoded file for each new version of NI-DAQmx? Does that include patches and minor versions?
Yes. 

Does this update policy align with the one used for NI Update Service on Windows?
No. My understanding is that we don't have to align it, I can double check and confirm.

In the long run, should this metadata be on ni.com instead?
No. @zhindes ?

Reply:
> In the long run, should this metadata be on ni.com instead?

Should it? Yes. Will it? No in the short-term, probably no in the long-term. Working with IT on this has been a really really big pain. Eventually we gave up and have resorted to this. A better solution is to leverage the NIPM data that already can do something like this. There are two problems:
* They currently only have a .NET assembly available to use their API, and they don't want to open up the backend impl in a way that customers can see it.
* It only works for Windows NIPM, not Linux. Linux might have some well-defined ni.com format for where we can find the package metadata, but I'd rather not rely on ni.com being stable, sadly.

So for now, keeping this metadata here and updating it when its useful to do so (new hardware is added probably) is what we're gonna go with.


>  Installed NI-DAQmx version (23.8.0) is equivalent or newer than the known version to install, (23, 8, 0)".

I can't think of a better phrase than "known" version. I don't want to suggest the version we know about is the latest or anything in case we don't update it. Hmm.... Also, can you update that string to match the format of the first version?


```python
f"Installed NI-DAQmx version ({installed_version}) is equivalent or newer than the known version to install, {known_version[0]}.{known_version[1]}.{known_version[2}".

Reply:
> I can't think of a better phrase than "known" version.

"It is known." --Game of Thrones

@zhindes How about "Installed NI-DAQmx version (x.x.x) is up to date. (Expected y.y.y or newer.)"?

Reply:
Love it!

Reply:
@bkeryan @zhindes : Is this fine ? 

![image](https://github.com/ni/nidaqmx-python/assets/927562/0add48f4-b2e2-41b8-b650-5d68826c536d)

Reply:
Leave a space between the period at the end of a sentence and the beginning of the next sentence.

y.y.y is a placeholder for the actual version, not something you should print.